client/src/components/PledgItem/PledgeItem.js

```diff
@@ -1,13 +1,10 @@
 import React from 'react'
 import './style.css'
 const PledgeItem = props => {
-  const { element } = props
-  //should git history to be 
-  // const { element, history } = props
-
+  const { element, history } = props
   return (
     <div className="pledge-div">
-      <img onClick={() => { console.log("history.push('')") }} alt={element.title} className="pledge-div__img" src={element.img} />
+      <img onClick={() => { history.push(`/action-category/pledge/${element.title}`, { pledge_id: element.pledge_id }) }} alt={element.title} className="pledge-div__img" src={element.img} />
       <p className="pledge-div__title">{element.title}</p>
```

Comment:
this element <p className="pledge-div__title">{element.title}</p>
needing to fix it's style it's not presented in the right way in the page, fix it's style